cmd/minishift/cmd/root.go

```diff
@@ -71,6 +74,15 @@ var RootCmd = &cobra.Command{
 
 		ensureConfigFileExists(constants.ConfigFile)
 
+		// Create MACHINE_NAME.json
+		jsonDataPath := filepath.Join(constants.Minipath, "machines", constants.MachineName+".json")
+		instanceConfig.Config, err = instanceConfig.NewInstanceConfig(jsonDataPath)
```

Comment:
why can't you use `instanceConfig.Config, err := instanceConfig.NewInstanceConfig(jsonDataPath)` ?